backend/src/main/java/com/github/EazyErik/service/DrinksService.java

```diff
@@ -1,13 +1,26 @@
 package com.github.EazyErik.service;
 
+import com.github.EazyErik.datalayer.Drink;
 import com.github.EazyErik.repository.DrinksRepository;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
+import java.util.List;
+import java.util.Optional;
+
 @Service
 @RequiredArgsConstructor
 public class DrinksService {
 
     private final DrinksRepository drinksRepository;
 
+
+    public Drink addToFavorite(Drink drinkToAdd,String username) {
+        drinkToAdd.setUsername(username);
+        return drinksRepository.save(drinkToAdd);
+    }
+
+    public List<Drink> getAllMyFavourites(String username) {
+        return drinksRepository.findAllByUsername(username);
+    }
```

Comment:
I think the Other way around, if u save drinkId's in the User Object, would make more sense.
Or do Other Users wanna know who else favorised a drink?